src/libs/async_file_loader.js

```diff
@@ -1,14 +1,10 @@
-/* global FileLoader */
 
-async function AsyncFileLoader(uri, options = {}) {
+async function AsyncFileLoader(uri) {
   return new Promise(resolve => {
-    const fileLoader = new FileLoader(uri);
-    fileLoader.onLoad = () => {
-      resolve();
-    };
-    if (options.onProgress) {
-      fileLoader.onProgress = this.onProgress;
-    }
+    const sc = document.createElement('script');
+    sc.src = uri;
+    document.body.appendChild(sc);
+    sc.onload = resolve;
```

Comment:
Wouldn't it be safer to set `onload` before calling `appendChild` ?

Reply:
Indeed. I changed it.